resources/gen-proton-blocklist.sh

```diff
@@ -14,7 +14,7 @@ for proton in "${!PROTON_BLOCK_LIBS[@]}"; do
     for block_lib in ${PROTON_BLOCK_LIBS[$proton]}; do
         for arch_suffix in "64" ""; do
             for wine_bin in "wine${arch_suffix}" "wine${arch_suffix}-preloader"; do
-                echo "${dist_dir}/bin/${wine_bin} ${dist_dir}/lib${arch_suffix}/${block_lib}"
+                echo "*(*/)${dist_dir}/bin/${wine_bin} *(*/)${dist_dir}/lib${arch_suffix}/${block_lib}"
```

Comment:
What are these `*(*/)`?

Reply:
GNU extension for fnmatch. The first star means "zero or more"

Reply:
Are these required now and path component without wildcards won't match?

Reply:
Yes, pattern match is now pure fnmatch. It allows wildcards at beginning and end of pattern.

Reply:
Then I guess the the config-generator shouldn't escape backslashes.